src/components/QuestionList/index.js

```diff
@@ -1,30 +1,41 @@
 import { h } from 'preact';
 import { Toggle } from '../Toggle';
 
-const QuestionList = ({ questions, updateQuestion, updateQuestionStatus }) =>
+const QuestionList = ({
+  questions,
+  updateQuestion,
+  updateQuestionStatus,
+  deleteQuestion,
+}) =>
   !questions.length ? null : (
     <div>
       <h2 class="center">Your questions</h2>
       {questions.map((question, index) => (
         <p class="edit-question">
-          <label>Q{index + 1}</label>
+          <div class="edit-question-controls">
+            <label>Q{index + 1}</label>
+            {updateQuestionStatus && (
+              <Toggle
+                label="Active"
+                on={question.status === 'live'}
+                onToggle={() =>
+                  updateQuestionStatus(
+                    question.slug,
+                    question.status === 'live' ? 'draft' : 'live'
+                  )
+                }
+              />
+            )}
+            {deleteQuestion && (
+              <a onClick={() => deleteQuestion(question.slug)}>&times;</a>
```

Comment:
Instead of an anchor, this can be a `<button type="button">`, perhaps with an `aria-label="Delete the question, ${question}`?